main.go

```diff
@@ -119,3 +122,15 @@ func getCommands() []cli.Command {
 		},
 	}
 }
+
+func execDiagnostics() {
```

Comment:
After rethinking about this, I suggest we think about this mechanism again, and try coming out with a different approach.